docs/md/usage-retry.md

```diff
@@ -57,6 +70,10 @@ and cannot yet submit a subset of a snapshot.
 So retrying any workflow from a workload snapshot
 will resubmit all entities from that snapshot.
 
+(Because of this, the optional workflow status filter is purely decorative:
+all sibling workflows from the same submission will be resubmitted,
+regardless of their status.)
```

Comment:
:-(

Reply:
Ditto.

I thought about removing status entirely as a retry input for this reason, but ultimately we hope to use it when we can submit snapshot subsets.

I could still do this as a one-liner (plus documentation updates), by keeping the underlying implementation but removing it from the retry request body spec.

https://github.com/broadinstitute/wfl/blob/develop/api/src/wfl/api/spec.clj#L21

What do you think?

Reply:
I say leave it as a "latent feature".